test/integration/scheduler/queue_test.go

```diff
@@ -295,7 +295,7 @@ func TestCoreResourceEnqueue(t *testing.T) {
 			wantRequeuedPods: sets.New("pod1"),
 		},
 		{
-			name:         "Pod got resource scaled down requeued to activeQ",
+			name:         "Pod rejected by the NodeResourcesFit plugin is requeued when the Pod is updated to scale down",
```

Comment:
I was going to add a case that a Pod resources are scaled down, but it already existed here.
I modified the name to make it clearer that it is a test for NodeResourcesFit.